frontend/public/locales/ja/monitoring.json

```diff
@@ -0,0 +1,38 @@
+{
+  "View in Metrics": "メトリクスでの表示",
+  "Name": "名前",
+  "Firing alerts": "アラートの実行",
+  "State": "状態",
+  "Creator": "作成者",
+  "Alerts": "アラート",
+  "Alert details": "アラートの詳細",
+  "Severity": "重大度",
+  "Description": "説明",
+  "Summary": "概要",
+  "Message": "メッセージ",
+  "Source": "Source",
```

Comment:
`Source` should be translated

Reply:
This has been updated in the third commit (latest batch of Japanese files).